includes/builder/class-boldgrid-editor-builder.php

```diff
@@ -134,6 +134,14 @@ public static function get_theme_buttons() {
 		global $boldgrid_theme_framework;
 
 		if ( empty( $boldgrid_theme_framework ) ) {
+			/**
+			 * Allow 3rd-Party Themes to add buttons
+			 *
```

Comment:
I see the "what", but can you add a "why"? e.g. Why do 3rd party themes need to do things differently?

Reply:
@bwmarkle updated for you